roles/provision_vcenter/templates/vcsa_on_esxi.j2

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I see those files are very similar, do you think it would be possible have just one, and parametrize via variable only the specifc inlude, which is different?

Reply:
thats a really good idea. Ive updated the templates....